pkg/api/vlabs/validate.go

```diff
@@ -363,6 +363,12 @@ func (a *Properties) validateMasterProfile() error {
 		}
 	}
 
+	if a.OrchestratorProfile.OrchestratorType == Kubernetes {
+		if m.IsVirtualMachineScaleSets() && m.VnetSubnetID != "" && m.FirstConsecutiveStaticIP != "" {
```

Comment:
nit:

"when masterProfile's availabilityProfile is VirtualMachineScaleSets and a vnetSubnetID is specified, the firstConsecutiveStaticIP should be empty and will be determined by an offset from the first IP in the vnetCidr"

Reply:
done